QuasarHelios2015/src/org/team1540/quasarhelios/AutonomousModeBase.java

```diff
@@ -66,7 +66,7 @@ protected void setClampOpen(boolean value) throws InterruptedException, Autonomo
     }
 
     protected void setClampHeight(float value) throws AutonomousModeOverException, InterruptedException {
-        Clamp.height.set(value);
+        Clamp.heightOrSpeed.set(value);
```

Comment:
Maybe we should set the mode so that `setClampHeight` will actually set the height.